cpp/benchmarks/text/subword.cpp

```diff
@@ -18,19 +18,22 @@
 #include <benchmarks/synchronization/synchronization.hpp>
 
 #include <cudf_test/column_wrapper.hpp>
+#include <cudf_test/file_utilities.hpp>
 
 #include <cudf/strings/strings_column_view.hpp>
+
 #include <nvtext/subword_tokenize.hpp>
 
 #include <filesystem>
 #include <fstream>
 #include <iostream>
 #include <vector>
 
+temp_directory const subword_tmpdir{"cudf_gbench"};
```

Comment:
question: Does this need to be global? 

suggestion: Seems this is only used in one static function, so maybe move the variable inside that function?